flink-core/src/main/java/org/apache/flink/types/RowKind.java

```diff
@@ -52,5 +52,26 @@
 	/**
 	 * Deletion operation.
 	 */
-	DELETE
+	DELETE;
+
+	/**
+	 * Returns a short string representation of this {@link RowKind}.
+	 *
+	 * <ul>
+	 *     <li>"I" represents {@link #INSERT}.</li>
+	 *     <li>"UB" represents {@link #UPDATE_BEFORE}.</li>
+	 *     <li>"UA" represents {@link #UPDATE_AFTER}.</li>
+	 *     <li>"D" represents {@link #DELETE}.</li>
+	 * </ul>
+	 */
+	public String shortString() {
```

Comment:
What if we add a parameter for each enum, for example, INSERT("I"), UPDATE_BEFORE("UB"). Then we don't need this switch-case?